app/javascript/components/patient/laboratory/FirstPositiveLaboratory.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import { PropTypes } from 'prop-types';
+import { Button, Form } from 'react-bootstrap';
+import moment from 'moment';
+
+import LaboratoryModal from './LaboratoryModal';
+
+class FirstPositiveLaboratory extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showModal: false,
+    };
+  }
+
+  render() {
+    return (
+      <React.Fragment>
+        {this.props.lab ? (
+          <div className={this.props.displayedLabClass || 'my-2'}>
+            <div className="first-positive-lab-result-header">
+              <Form.Label className="input-label mb-1">FIRST POSITIVE LAB RESULT</Form.Label>
+              <div className="edit-link">
+                <Button variant="link" id="delete-first_positive_lab" className="py-0 px-1 icon-btn-dark" onClick={() => this.props.onChange(null)}>
+                  <i className="fas fa-times fa-fw"></i>
+                </Button>
+              </div>
+              <div className="edit-link">
+                <Button variant="link" id="edit-first_positive_lab" className="py-0 px-1 icon-btn-dark" onClick={() => this.setState({ showModal: true })}>
+                  <i className="fas fa-edit fa-fw"></i>
+                </Button>
+              </div>
+            </div>
+            <div>
+              <span className="first-positive-lab-result-field-name">Type: </span>
+              <span>{this.props.lab.lab_type || '--'}</span>
+            </div>
+            <div>
+              <span className="first-positive-lab-result-field-name">Specimen Collection Date: </span>
+              <span>{this.props.lab.specimen_collection ? moment(this.props.lab.specimen_collection, 'YYYY-MM-DD').format('MM/DD/YYYY') : '--'}</span>
+            </div>
+            <div>
+              <span className="first-positive-lab-result-field-name">Report Date: </span>
+              <span>{this.props.lab.report ? moment(this.props.lab.report, 'YYYY-MM-DD').format('MM/DD/YYYY') : '--'}</span>
+            </div>
+            <div>
+              <span className="first-positive-lab-result-field-name">Result: </span>
+              <span>{this.props.lab.result || '--'}</span>
+            </div>
+          </div>
+        ) : (
+          <div className="mb-2">
+            <div>
```

Comment:
yep, cuz otherwise this happens

![Screen Shot 2021-06-17 at 1 59 21 PM](https://user-images.githubusercontent.com/9956561/122449811-47a3b880-cf74-11eb-8063-2b502525ad79.png)

Reply:
so this shouldn't even be a form label because it's not associated with an input (508 compliance). I'd change this to a div in both places and just add the necessary styling 

![Screen Shot 2021-06-18 at 7 49 41 AM](https://user-images.githubusercontent.com/35042815/122556728-dadafd80-d009-11eb-9759-7e695cc0fe60.png)

Reply:
`<div className="input-label mb-2">POSITIVE LAB RESULT</div>` should do the trick

Reply:
https://github.com/SaraAlert/SaraAlert/pull/861/commits/a79a65940f2a0383f7f4d8b28330d2b7e1db30ae